app/models/katello/repository.rb

```diff
@@ -121,6 +121,7 @@ class Repository < Katello::Model
     before_validation :set_container_repository_name, :unless => :skip_container_name?
 
     scope :has_url, -> { joins(:root).where.not("#{RootRepository.table_name}.url" => nil) }
+    scope :not_uln, -> { joins(:root).where("#{RootRepository.table_name}.url NOT LIKE 'uln://%'") }
```

Comment:
Most places in code we only use `start_with?('uln')` and here we have uln:// .. Not really important but we could make it consistent.